src/stores/routePathStore.ts

```diff
@@ -14,23 +14,30 @@ export enum AddLinkDirection {
     AfterNode,
 }
 
+export enum RoutePathViewTab {
```

Comment:
Enum vs string literal type - You decide :)

Reply:
Enum :)